code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -336,7 +337,8 @@
 	w_class = ITEMSIZE_NORMAL
 	slot_flags = null
 	max_stack = 1
-	reload_sound = 'sound/weapons/reload_shell.ogg'
+	reload_sound = 'sound/weapons/reloads/shotgun_pump.ogg'
+	drop_sound = /decl/sound_category/generic_drop_sound
```

Comment:
Shouldn't this be /decl/sound_category/casing_drop_sound_shotgun?

This might be the reason why the shotgun shells sound weird. Please remove  var/casing_drop_sound.